css-layout-classes/index.html

```diff
@@ -60,9 +57,9 @@ <h3>Super Smash Bros. Ultimate</h3>
 
       <div class="row">
 
-        <div class="container" id="container-three">
+        <div class="col-fourth" id="row-three">
```

Comment:
Remove the id. This `div` is a column, not a row, so this is just adding confusion